<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Dialog::DlgSettingsDrawStyles</class>
 <widget class="QWidget" name="Gui::Dialog::DlgSettingsDrawStyles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Draw styles</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0" rowminimumheight="0,0,0,0,0,0,0,0">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Selected hidden line pattern</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxSelectionOnTop">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Show selection on top</string>
          </property>
          <property name="prefEntry" stdset="0">
           <string>ShowSelectionOnTop</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Gui::PrefDoubleSpinBox" name="spinLineWidthMultiplier">
          <property name="toolTip">
           <string>Muplication factor to increase the width the selected line.</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>SelectionLineThicken</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="Gui::PrefDoubleSpinBox" name="spinSelectionHiddenLineWidth">
          <property name="toolTip">
           <string>Width of the hidden line.</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>SelectionHiddenLineWidth</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Line width multiplier</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Transparency</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboLinePattern"/>
        </item>
        <item row="1" column="1">
         <widget class="Gui::PrefDoubleSpinBox" name="spinTransparencyOnTop">
          <property name="toolTip">
           <string>Transparency for the selected object when being shown on top.</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>TransparencyOnTop</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Selected hidden line width</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxSelection">
     <property name="title">
      <string>Hidden Lines</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="_3" columnstretch="0,0">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxFaceColor">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Enable preselection and highlight by specified color</string>
          </property>
          <property name="text">
           <string>Override face color</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>HiddenLineOverrideFaceColor</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Transparency</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Gui::PrefColorButton" name="FaceColor">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="color" stdset="0">
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>HiddenLineFaceColor</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxLineColor">
          <property name="toolTip">
           <string>Enable selection highlighting and use specified color</string>
          </property>
          <property name="text">
           <string>Override line color</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>HiddenLineOverrideColor</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="Gui::PrefDoubleSpinBox" name="spinTransparency">
          <property name="toolTip">
           <string>Override transparency of all objects in the scene.</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>HiddenLineTransparency</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="Gui::PrefColorButton" name="LineColor">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="color" stdset="0">
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>HiddenLineColor</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxShaded">
          <property name="text">
           <string>Shaded</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
          <property name="prefEntry" stdset="0">
           <string>HiddenLineShaded</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxBackground">
          <property name="text">
           <string>Override background color</string>
          </property>
          <property name="prefEntry" stdset="0">
           <string>HiddenLineOverrideBackground</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Gui::PrefColorButton" name="BackgroundColor">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="color" stdset="0">
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>HiddenLineBackground</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxSelection_2">
     <property name="title">
      <string>Shadow</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="_4" columnstretch="0,0" columnminimumwidth="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="5" column="1">
         <widget class="Gui::PrefDoubleSpinBox" name="spinBoxGroundScale">
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
          <property name="prefEntry" stdset="0">
           <string>ShadowGroundScale</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Light color</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxFlatLines">
          <property name="text">
           <string>Flat lines</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
          <property name="prefEntry" stdset="0">
           <string>ShadowFlatLines</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="Gui::PrefColorButton" name="GroundColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="color" stdset="0">
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>ShadowGroundColor</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ground color</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxShowGround">
          <property name="text">
           <string>Show ground</string>
          </property>
          <property name="prefEntry" stdset="0">
           <string>ShadowShowGround</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Light intensity</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="Gui::PrefColorButton" name="LightColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="color" stdset="0">
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="prefEntry" stdset="0">
           <cstring>ShadowLightColor</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>View</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Gui::PrefDoubleSpinBox" name="spinBoxLightIntensity">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.800000000000000</double>
          </property>
          <property name="prefEntry" stdset="0">
           <string>ShadowLightIntensity</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Ground scale</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="Gui::PrefCheckBox" name="checkBoxSpotLight">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Use spot light</string>
          </property>
          <property name="prefEntry" stdset="0">
           <string>ShadowSpotLight</string>
          </property>
          <property name="prefPath" stdset="0">
           <string>View</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="spacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>183</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="spacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::ColorButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Widgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefColorButton</class>
   <extends>Gui::ColorButton</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefCheckBox</class>
   <extends>QCheckBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBoxFaceColor</tabstop>
  <tabstop>FaceColor</tabstop>
  <tabstop>checkBoxLineColor</tabstop>
  <tabstop>LineColor</tabstop>
  <tabstop>spinTransparency</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxFaceColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>FaceColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLineColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>LineColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
